src/app/payment/Payment.tsx

```diff
@@ -462,6 +462,16 @@ export function mapToPaymentProps({
 
     let selectedPaymentMethod;
     let filteredMethods;
+
+    // Prevent a payment method from being rendered
+    remove(methods, method => {
```

Comment:
I wonder if instead of using remove is better to use `filteredMethods` and use filter instead? Not sure if modifying the original value can cause issues.